src/module/comments.js

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable consistent-return */
+/* eslint import/no-cycle: 0 */
+// eslint-disable-next-line no-unused-vars
+// import { text } from '@fortawesome/fontawesome-svg-core';
+import SHOWS from './home.js';
+import closeIcon from '../Assets/closeIcon.png';
+
+const baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi';
+const appsEndPoint = '/apps/';
+const commentsEndPoint = '/comments/';
+const CommentEndPt = '/comments?item_id=';
+const id = 'LHFUkzcDvsXW9NARtYgJ';
+const popUpOverlay = document.querySelector('.popup-overlay');
+
+const comment = {
+  item_id: '',
+  username: '',
+  comment: '',
+};
+
+let submitbtn;
+
+export default class COMMENTS {
+  static createNewComment = () => {
+    submitbtn.addEventListener('click', async (e) => {
+      const input = document.querySelector('.input');
+      const cmt = document.querySelector('textarea.comment');
+      if (input.value !== '' && cmt.length !== '') {
+        comment.username = input.value;
+        comment.comment = cmt.value;
+        comment.item_id = e.target.id.toString();
+        try {
+          const response = await fetch(baseUrl + appsEndPoint + id + commentsEndPoint, {
+            method: 'POST',
+            headers: {
+              'Content-Type': 'application/json',
+            },
+            body: JSON.stringify(comment),
+          });
+          const data = await response.text();
+          // Display the new comment on the screen
+          const commentList = document.querySelector('.commentors');
+          const newComment = `<li>${comment.creation_date} ${comment.username}: ${comment.comment}</li>`;
+          commentList.insertAdjacentHTML('beforeend', newComment);
+          // Update the comment count
+          const commentCount = document.querySelector('#comments-count');
+          commentCount.textContent = JSON.parse(data).length;
```

Comment:
- [ ] I see that you are trying to update the comments count after the comment has been created. Unfortunately, it's not working as expected - the comment count remains the same even after a comment has been added. However, if you reload the page, and navigate to the movie again, then the comment count displays correctly.
Kindly debug this 👍